rrd/.gitignore

```diff
@@ -1,4 +0,0 @@
-# Ignore everything in this directory
```

Comment:
Is it expected to touch rrd/.gitignore ?

Reply:
no, stupid thing...

Reply:
Fixed.